components/form/builder/README.md

```diff
@@ -0,0 +1,23 @@
+# FormBuilder
+
+> Description
+
+<!-- ![](./assets/preview.png) -->
+
+## Installation
+
+```sh
+$ npm install @schibstedspain/sui-form-builder --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import FormBuilder from '@schibstedspain/sui-form-builder'
+
+return (<FormBuilder />)
+```
+
+
+> **Find full description and more examples in the [demo page](#).**
```

Comment:
Not enough documentation, you must document required props at least.

Reply:
Yep, I'm just create the README

Reply:
DONE!